src/timelines.js

```diff
@@ -483,6 +484,10 @@ export default Kapsule({
         .direction('s')
         .offset([5, 0])
         .html(d => {
+          if (state.segmentTooltipHtml) {
+            return state.segmentTooltipHtml(d);
```

Comment:
After your other PR #76 is in, it probably makes more sense to replace this with `segmentTooltipHtml(d.data)`.